wire-ios-sync-engine/Source/UserSession/ZMUserSession/ZMUserSession.swift

```diff
@@ -1031,6 +1030,12 @@ extension ZMUserSession: ZMSyncStateDelegate {
             self?.delegate?.authenticationInvalidated(error as NSError, accountId: accountId)
         }
     }
+
+    func checkE2EICertificateExpiryStatus() {
+        if e2eiFeature.isEnabled {
+            NotificationCenter.default.post(name: .checkForE2EICertificateExpiryStatus, object: nil)
+        }
```

Comment:
```suggestion
        guard e2eiFeature.isEnabled else { return }
        NotificationCenter.default.post(name: .checkForE2EICertificateExpiryStatus, object: nil)
```
Usually it's easier to read code with less indentation and follow have the golden path idea.